From: Jiri Kosina <jkosina@suse.cz>
Subject: jump_label: export to modules
Patch-mainline: Never, SUSE-specific, upstream has static keys done differently (and exported)
References: bsc#1089343 CVE-2018-3646

We need jump labels for dynamically enabling/disabling L1D flushes in
kvm-intel module.

Signed-off-by: Jiri Kosina <jkosina@suse.cz>
---
 kernel/jump_label.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/kernel/jump_label.c
+++ b/kernel/jump_label.c
@@ -71,6 +71,7 @@ void jump_label_inc(struct jump_label_ke
 	atomic_inc(&key->enabled);
 	jump_label_unlock();
 }
+EXPORT_SYMBOL_GPL(jump_label_inc);
 
 void jump_label_dec(struct jump_label_key *key)
 {
@@ -80,6 +81,7 @@ void jump_label_dec(struct jump_label_ke
 	jump_label_update(key, JUMP_LABEL_DISABLE);
 	jump_label_unlock();
 }
+EXPORT_SYMBOL_GPL(jump_label_dec);
 
 static int addr_conflict(struct jump_entry *entry, void *start, void *end)
 {
